modules/01-mods.js

```diff
@@ -487,19 +499,24 @@ async function slashModSlowmode(interaction) {
 async function slashModSummary(interaction) {
   await interaction.deferReply({ ephemeral: true });
   const member = interaction.options.getMember("user");
-  const time = interaction.options.getInteger("history");
+  const time = interaction.options.getInteger("history") ?? 28;
 
   const data = await Module.db.infraction.getSummary(member.id, time);
   const response = [`**${member}** has had **${data.count}** infraction(s) in the last **${data.time}** day(s), totaling **${data.points}** points.`];
   if ((data.count > 0) && (data.detail.length > 0)) {
     for (const record of data.detail) {
       const mod = interaction.guild.members.cache.get(record.mod) || `Unknown Mod (<@${record.mod}>)`;
-      response.push(`${record.timestamp.toLocaleDateString()} (${record.value} pts, modded by ${mod}): ${record.description}`);
+      let pointsPart = record.value === 0 && mod.id !== Module.client.id ? "Note" : `${record.value} pts`;
+      response.push(`\`${record.timestamp.toLocaleDateString()}\` (${pointsPart}, modded by ${mod}): ${record.description}`);
     }
   }
-  const modlogs = await interaction.guild.channels.cache.get(sf.channels.modlogs);
-  await modlogs.send(response.join("\n"), { split: true });
-  await interaction.deferReply({ content: `I've put the summary you requested in ${modlogs}.`, ephemeral: true });
+
+  await interaction.editReply({ embeds: [
+    u.embed({ author: interaction.member })
+    .setTitle("Infraction Summary")
+    .setDescription(response.join("\n"))
```

Comment:
We need to make sure this fits. We don't have to do a fancy job of it, just chop off the end and put [. . .] if it's too much.